### PR TITLE
Manually bump net.minidev:json-smart from 2.4.11 to 2.5.0

### PR DESCRIPTION
**Pull request by scrawfor99**
### Description
 Manually update net.minidev:json-smart from 2.4.11 to 2.5.0 since #8569 failed. 

### Check List
- [ ] ~New functionality includes testing.~
  - [ ] ~All tests pass~
- [ ] ~New functionality has been documented.~
  - [ ] ~New functionality has javadoc added~
- [x] Commits are signed per the DCO using --signoff
- [x] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by scrawfor99** [comment]
Failure seems like a flaky result from Jenkins--will manually trigger retry but pushing text change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,9 +40,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `org.bouncycastle:bcprov-jdk15on` to `org.bouncycastle:bcprov-jdk15to18` version 1.75 ([#8247](https://github.com/opensearch-project/OpenSearch/pull/8247))
 - Bump `org.bouncycastle:bcmail-jdk15on` to `org.bouncycastle:bcmail-jdk15to18` version 1.75 ([#8247](https://github.com/opensearch-project/OpenSearch/pull/8247))
 - Bump `org.bouncycastle:bcpkix-jdk15on` to `org.bouncycastle:bcpkix-jdk15to18` version 1.75 ([#8247](https://github.com/opensearch-project/OpenSearch/pull/8247))
+- Bump `net.minidev:json-smart` from 2.4.11 to 2.5.0 ([#8576](https://github.com/opensearch-project/OpenSearch/pull/8576))
```
Comment (owaiskazi19) [inline CHANGELOG.md:43]: Change needs to be under `Unreleased 2.x`

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,9 +40,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 - Bump `com.google.cloud:google-cloud-core-http` from 2.17.0 to 2.21.0 ([#8586](https://github.com/opensearch-project/OpenSearch/pull/8586))
 
 
+
```
Comment (owaiskazi19) [inline CHANGELOG.md:47]: Nit: Extra line

**Comment by owaiskazi19** [comment]
@scrawfor99 can you take care of the conflicts?

**Comment by scrawfor99** [comment]
@owaiskazi19 should be all set!

**Comment by owaiskazi19** [comment]
> @owaiskazi19 should be all set!

@scrawfor99 the changes are still under `Unreleased 3.0`. We have included it in Under 2.x [here](https://github.com/opensearch-project/OpenSearch/pull/8604/files). Let's do the same in this PR as well.

**Review by owaiskazi19 — APPROVED** [review]

**Review by Poojita-Raj — APPROVED** [review]

**Comment by owaiskazi19** [comment]
@scrawfor99 can you backport this to 2.x?

**Pull request merged**